openasip/src/applibs/LLVMBackend/RISCVTDGen.cc

```diff
@@ -49,14 +49,30 @@
 
 #define DEBUG_RISCV_TDGEN 0
 
-RISCVTDGen::RISCVTDGen(const TTAMachine::Machine& mach)
+RISCVTDGen::RISCVTDGen(const TTAMachine::Machine& mach, bool roccEn)
     : TDGen(mach, false), bem_(NULL) {
-    bem_ = BEMGenerator(mach).generate();
+    bem_ = BEMGenerator(mach, roccEn).generate();
     assert(bem_ != NULL);
     findCustomOps();
     initializeBackendContents();
 }
 
+std::string
```

Comment:
These methods should be removed. They are not used and have crept in somehow to this commit.